### PR TITLE
corrected typo in contributing.md

### PR DESCRIPTION
**Pull request by SwRaw**

### CODE REVIEW

**Review by Rmalavally — APPROVED** [review]

**Comment by Rmalavally** [comment]
@saadrahim I am not authorized to do so. Cannot squash and merge.

**Comment by SwRaw** [comment]
conflict resolved.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,10 +38,28 @@ as follows:
 
 ## Filenames and folder structure
 
-Snake case is preferred for file names. Our documentation follows pitchfork for
-folder structure. All documentation is in `/docs` except for special files like
-the contributing guide in the `/` folder. All images used in the documentation are
-place in the `/docs/data` folder.
+Please use snake case for file names. Our documentation follows pitchfork for
```
Comment (Rmalavally) [inline CONTRIBUTING.md:41]: Could you add an example?

Using snake case refers to adding an underscore(_) character in each space and the first letter of each word is written in lowercase. 
For example, 'use_underscore_and_lower_case'

**Review by Rmalavally — DISMISSED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -38,10 +38,28 @@ as follows:
...
+
+All contributions to ROCm documentation should arrive via the
+[GitHub Flow](https://docs.github.com/en/get-started/quickstart/github-flow)
+targetting the develop branch of the repository. If you are unable to contribute
+via the GitHub Flow, feel free to email us. TODO, confirm email address.
+
+### GitHub Issue
+
+Issues on existing or absent docs can be filed as [GitHub issues
+](https://github.com/RadeonOpenCompute/ROCm/issues).
```
[the remark this answers is not in the document]
Comment (SwRaw) [reply CONTRIBUTING.md:60]: I have not written this document. I just came across a typo while quickly going through it and thought of fixing it.
As for your question, I think "as" is okay here. It says that issues can be filed as GitHub issues.

[the remark this answers is not in the document]
Comment (Rmalavally) [reply CONTRIBUTING.md:60]: Sounds good. Thanks for clarifying, @SwRaw.

File: CONTRIBUTING.md (hunk 1 above)
Comment (samjwu) [reply CONTRIBUTING.md]: Fixed in https://github.com/RadeonOpenCompute/ROCm/commit/6b4368e01cd0065eaee9e7d903b7f2bbc066e22b

**Review by samjwu — APPROVED** [review]

**Pull request merged**